evaluator/opa.go

```diff
@@ -51,11 +53,17 @@ var opaConfig = `{
 			"resource": "file://%s"
 		}
 	},
-	"decision_logs": {
-		"console": %t
-	}
+%s
```

Comment:
This is getting a bit hard to read now, and is a good use case for the [templates package](https://pkg.go.dev/text/template).